package.json

```diff
@@ -25,8 +25,10 @@
     "test:format": "prettier \"**/*.{tsx,ts,md}\" --check",
     "postinstall": "husky install",
     "precommit": "lint-staged",
-    "storybook": "start-storybook -p 6006 -s ./",
-    "build-storybook": "build-storybook"
+    "storybook": "start-storybook --port 6006 --static-dir ./static",
+    "storybook:build": "build-storybook --static-dir ./static",
+    "storybook:serve": "serve storybook-static",
```

Comment:
What do we need this for?

Reply:
To answer this question and your question in the summary at once: We need this to check Storybook build in the browser on our local machines. Gatsby brings this feature out of the box with `gatbsy serve`, but Storybook doesn't include a server. 
Otherwise you would need to install a webserver on your machine to verify the output folder.

Reply:
> Usually the development and build output should be the same, but since different things happen there, sometimes the dev server works and the build doesn't.

Does that happen with Storybook? Has it happened to you in the past? I'm not convinced that we need to do this for Storybook...

I'd rather use `npx serve storybook-static` when I need to debug something instead of having a solid dependency to `serve`. All in all I'm fine with this, either way. It certainly doesn't do any harm. But it's yet another dependency that we will probably need and use only once in a leap year.